students/kegan/session03/mailroom.py

```diff
@@ -0,0 +1,301 @@
+#!/usr/bin/env python3
+
+"""
+Kathryn Egan
+
+This program manages donors and their donations for
+Miuvenile Care, a charity organizaiton for disadvantaged
+kittens.
+"""
+
+
+DONORS = {
+    'Benedict Cumberbatch': [200000.0],
+    'Elon Musk': [10000.0, 150000.0, 100000.0],
+    'Dad': [20.0, 5.0],
+    'Donald Trump': [2.81],
+    'Billy Neighbor': [.54, .01, .25]}
+
+COLUMN_WIDTHS = [0] * 4
+
+
+def main():
+    """ Module that drives main menu."""
+    options = {
+        '1': {
+            'prompt': 'Send a Thank You',
+            'module': print_thank_you},
+        '2': {
+            'prompt': 'Create a Report',
+            'module': print_report},
+        '3': {
+            'prompt': 'Write Thank Yous',
+            'module': write_thank_yous},
+        '4': {
+            'prompt': 'Quit',
+            'module': exit_program}}
+    while True:
+        print('Please choose from the following options:')
+        prompt = '\n'.join(['{} {}'.format(
+            option, options[option]['prompt']) for option in options])
```

Comment:
really slick!

one hint -- if you want both the key and the value in a dict, you can use items():

`['{} {}'.format( option_num, option['prompt']) for option_num, option in options.items()]`